magpie/api/management/user/user_views.py

```diff
@@ -99,7 +100,14 @@ def assign_user_group(request):
     """Assign a user to a group."""
     db = request.db
     user = get_user_matchdict_checked_or_logged(request)
-    group = get_group_matchdict_checked(request)
+
+    group_name = get_value_multiformat_post_checked(request, 'group_name')
+    group = evaluate_call(lambda: zig.GroupService.by_group_name(group_name, db_session=request.db),
```

Comment:
I don't see where this `zig` namespace is defined. Where is it coming from?

Reply:
It's actually defined in api_requests.py... `zig` is used in other places in `user_views.py` so I used the same reference here. I could remove the `zig.` part and it works just fine. I think we could talk about cleaning these imports in another issue/PR, but for now I just used the same reference as other places in the same file.